lib/peg.d.ts

```diff
@@ -308,21 +304,21 @@ type Problem = [
 /** Thrown if the grammar contains a semantic error. */
 export class GrammarError extends Error {
   /** Location of the error in the source. */
-  location?: LocationRange;
+  public location?: LocationRange;
```

Comment:
Added a lint rule that said that all fields and methods needed explicit visibility, which I like.  Kept these public to not have a breaking change.